pulsar-broker/src/main/java/org/apache/pulsar/broker/service/schema/SchemaRegistry.java

```diff
@@ -41,6 +41,8 @@
 
     CompletableFuture<SchemaVersion> deleteSchemaStorage(String schemaId);
 
+    CompletableFuture<SchemaVersion> deleteSchemaStorage(String schemaId, boolean forcefully);
```

Comment:
mark this a default method?

Reply:
that will be tricky because `deleteSchemaStorage(schemaId)` calls `deleteSchemaStorage(schemaId, false)` . and we can't define default behavior of `deleteSchemaStorage(schemaId, false)` by calling existing `deleteSchemaStorage(schemaId)` because it creates cyclic call.